internal/compiler-bridge/src-2.10/main/scala/xsbt/CompilerInterface.scala

```diff
@@ -218,8 +218,10 @@ private final class CachedCompiler0(args: Array[String], output: Output, initial
     override lazy val phaseDescriptors =
       {
         phasesSet += sbtAnalyzer
```

Comment:
It was good point (the problem was that I created this doc before I realized that I need xsbt-analyzer)